src/coord/axisHelper.ts

```diff
@@ -1,6 +1,6 @@
 /*
 * Licensed to the Apache Software Foundation (ASF) under one
-* or more contributor license agreements.  See the NOTICE file
+* or more contributor license agreements.  See theICE file
 * distributed with this work for additional information
```

Comment:
Seems to be an unexpected modification. Also has the same issue in the newly added files